### DISCUSSION TITLE
LocalRotation

### DISCUSSION THREAD

**Discussion by Nickykun** [nem0/LumixEngine#1490] [Ideas]
I don't understand why there is no Local Rotation when there is a Local Position.
![image](https://github.com/nem0/LumixEngine/assets/46878391/9248c51e-7587-44a1-8025-643d7c9cbe29)

**Comment by nem0** [comment]
I don't think there was any special reason. I just did not need it for my use case. But it can be easily exposed if necessary

**Comment by nem0** [comment]
done in fe87e5a347da36e5d246e8149e1d5cce234dbc15
